parseutils.c

```diff
@@ -596,8 +406,11 @@ get_double_from_file(char *ftr)
 	if (strcmp(rawstr, "max") == 0)
 		result = DBL_MAX;
 	else
+        #if PG_VERSION_NUM < 160000
 		result = float8in_internal(rawstr, NULL, "double precision", rawstr);
-
+        #else
+		result = float8in_internal(rawstr, NULL, "double precision", rawstr, NULL);
```

Comment:
Looks like if it's `NULL` then it just throws a normal error; the context is used for soft erroring, so won't abort but captures info about the exception.  I'd say this current usage is fine, since the previous version wouldn't be using soft error support.

Reply:
(see what eventually becomes `errsave_start()` via `errsave_domain()` and `ereport()`)

Reply:
Assuming you're ok with that response @cbandy, feel free to resolve this